modules/ROOT/pages/_partials/generic-configuring-dev-env.adoc

```diff
@@ -0,0 +1,55 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Configuring your Development Environment
+
+
+* [x] You have provisioned one of the {product-name} as described in xref:provisioning.adoc[Provisioning Services].
+* [x] You have associated the service with a {mobile-client} as described in xref:binding-mobile-app-record-to-mobile-services.adoc[Binding].
```

Comment:
I know this is the generic document, but not all services require a binding. Afaik, metrics just 'works' once it is deployed and will collect metrics from other services (keycloak, etc.). So this could be an optional step. What do you think?

Reply:
Makes sense, yes.